rofl-client/ts/README.md

```diff
@@ -88,7 +89,10 @@ type EthTx = {
   gas_limit: number;
   /** Hex address (0x optional). Empty string => contract creation. */
   to: string;
-  /** JSON number; must fit JS number range (backend expects u128). */
+  /**
+   * JSON number; must fit JS number range (backend expects u128).
```

Comment:
what does this mean "must fit JS number range"?

Reply:
Yes, have the fix in this PR https://github.com/oasisprotocol/oasis-sdk/pull/2393. will rebase and update this PR once that's merged